internal/website/gatherexamples/gatherexamples.go

```diff
@@ -14,22 +14,24 @@
 
 // Command gatherexamples extracts examples in a Go module into a JSON-formatted
 // object. This is used as input for building the Go CDK Hugo website.
-// Examples must include a comment that starts with "// This example is used in"
+//
+// Examples must include a comment
+// "// PRAGMA(gocloud.dev): Package this example for gocloud.dev."
 // somewhere in the function body in order to be included in this tool's output.
 //
 // gatherexamples does some minimal rewriting of the example source code for
 // presentation:
 //
 //   - Any imports the example uses will be prepended to the code.
 //   - log.Fatal(err) -> return err
```

Comment:
Why do this rewrite?

Reply:
I don't know for sure. I think it was because you can't `return err` in a Go example due to the signature, but more realistic code would do that.